modules/ducktests/tests/ignitetest/services/ignite.py

```diff
@@ -46,13 +46,42 @@ def __init__(self, context, config, num_nodes, jvm_opts=None, modules=None):
 
     # pylint: disable=W0221
     def start(self, timeout_sec=180):
+        self._rotate_log()
+
         super().start()
 
         self.logger.info("Waiting for Ignite(s) to start...")
 
         for node in self.nodes:
             self.await_node_started(node, timeout_sec)
 
+    def restart(self, timeout_sec=180):
+        """
+        Restart ignite cluster without cleaning.
+        """
+        self.stop()
+
+        self._rotate_log()
+
+        for node in self.nodes:
+            super().start_node(node)
+
+        self.logger.info("Waiting for Ignite(s) to start...")
+
+        for node in self.nodes:
+            self.await_node_started(node, timeout_sec)
+
+    def _rotate_log(self):
```

Comment:
Do we really need to rotate the log?
Why the same log usage is a problem?

Reply:
Because we do not append to stream, so log file will be broken.
But yes, we can do fix this just adding 1 character

Reply:
Now he is overwriting the log file.
I shoud it is more correct to rotate logs

Reply:
Rotation should be performed by log4j if necessary. Node restart is not a reason to rotate logs.

Reply:
log4j works on stdout, not log file